TRANSLATING.md

```diff
@@ -0,0 +1,225 @@
+
+#### With git command line tools
+
+The first step is to get the local repo up to date with the upstream master:
+
+```
+cd projectdir/jamulus
+git fetch upstream
+```
+
+This fetches information about the current state of the upstream repo. It is now necessary to apply any upstream changes to the local repo.
+As the user will not be updating the `master` branch themselves, `git rebase` can be used to fast-forward to the current state:
+
+```
+git checkout master
+git rebase upstream/master
+git push
+```
+
+Finally, create a new branch for the changes that will be done (do not just do them on `master`):
+
+```
+git checkout -b v370-german
+```
+
+(The branch name is chosen by the user; the above name v370-german is an example for the German translation of V3.7.0)
+
+The branch will be used later as the source of a Pull Request.
+
+#### With Github Desktop
+
+Select the current repository as `jamulus`, and the current branch as `master`.
+
+Click on "Fetch origin" to get fully up to date with upstream.
+
+Drop down the "Current Branch" menu and click on the **New Branch** button. Give the branch a name such as `v370-german` and click **Create Branch**.
+
+Do not click on **Publish branch** just yet.
+
+### Work on the translation file
+
+Open Qt Linguist, and navigate to the directory `src/res/translation` within your project directory.
+
+In this directory are translation source files for each language, each with a `.ts` suffix. Don't worry about the `.qm` files, as they are compiled when building the release code.
+
+Open the `.ts` file for the language being worked on.
+
+Each context in the left-hand column represents a source file or graphica form containing translatable strings. The icon beside each shows a green tick if there is nothing further to be done,
+a yellow tick if a translation needs attention, and a question mark if there are untranslated strings.
+
+After clicking on a Context, the list of strings needing translation is shown, with a green tick beside the ones that have been done. The right-hand pane will show either the source code or the form,
+so that the usage can be checked, in case this aids the translation.
+
+Strings with a grey tick can be ignored. They are old strings that used to be used but are no longer. They are retained for reference in case they should be used again in the future.
+
+Again, strings with a question mark require translation.
+
+After some or all of the required strings have been translated, the file can be saved.
+
+### Submit the updated translation as a PR
+
+Once all translations have been done, the branch containing the changes must be commited to your own repo, pushed to Github (`origin`) and then a Pull Request raised to the upstream repo.
+
+#### With git command line tools
+
+First, check the current branch with `git status`, to make sure it is correct, and then do a commit:
+
+```
+git commit -am 'Updated translations for v3.7.0'
```

Comment:
```suggestion
git commit -am 'Update German translations for v3.7.0'
```

Commit message should include the language and should be imperative-style (at least a quick scan of recent commit messages shows that this is the most common style).